rubicon/objc/objc.py

```diff
@@ -1314,6 +1326,65 @@ def __setattr__(self, name, value):
                 super(ObjCInstance, type(self)).__setattr__(self, name, value)
 
 
+class ObjCListInstance(ObjCInstance):
+    def __getitem__(self, item):
+        if item >= self.count:
+            raise IndexError('list index out of range')
+        return self.objectAtIndex(item)
+
+    def __setitem__(self, item, value):
+        try:
+            self.insertObject_atIndex_(value, item)
+        except AttributeError:
+            name = self.objc_class.name
+            raise TypeError("'%s' object does not support item assignment" % name)
+
+    def __len__(self):
+        return self.count
```

Comment:
I'm a bit confused why this works. If I'm not mistaken, #41 makes all properties behave as methods by default, and to access them using attribute syntax you need to declare them with `declare_property`. That would mean that by default you need to write `self.count()` to get the array's item count, because `self.count` would be the method object. Any ideas what is happening here @freakboy3742?

(The same question applies to `ObjCDictInstance.__len__` below.)